e2e/common.go

```diff
@@ -0,0 +1,67 @@
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"reflect"
+	"runtime/debug"
+	"testing"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+func notError(t *testing.T, e error) {
+	if e != nil {
+		t.Errorf("Error: %s", e)
+		debug.PrintStack()
+	}
+}
+
+func assertNotEqual(t *testing.T, want, got interface{}, msg string) {
+	if reflect.DeepEqual(want, got) {
+		t.Errorf("Wanted %s, got %s. %s", want, got, msg)
+		debug.PrintStack()
+	}
+}
+func assertEqual(t *testing.T, want, got interface{}, msg string) {
+	if !reflect.DeepEqual(want, got) {
+		t.Errorf("Wanted [%v], got [%v]. %s", want, got, msg)
+		debug.PrintStack()
+	}
+}
+
+func getPath(p string) string {
+	path, err := bazel.Runfile(p)
+	if err != nil {
+		panic(err)
+	}
+	return path
+}
+
+var bazelPath string
+var ibazelPath string
+
+const mainGoPath = "e2e/simple/main.go"
+const BUILDPath = "e2e/simple/BUILD.bazel"
+
+func init() {
+	bazelPath = getPath("e2e/bazel/bazel")
```

Comment:
Is there a reason you don't initialize these variables before this function is run? If so leave a comment